shard-operator/src/test/java/com/redhat/service/bridge/shard/operator/ManagerSyncServiceTest.java

```diff
@@ -51,7 +51,7 @@ public void testBridgesAreDeployed() throws JsonProcessingException, Interrupted
         String expectedJsonUpdateProvisioningRequest =
                 String.format("{\"id\": \"myId-1\", \"name\": \"myName-1\", \"endpoint\": \"myEndpoint\", \"customerId\": \"%s\", \"status\": \"PROVISIONING\"}", TestSupport.CUSTOMER_ID);
         String expectedJsonUpdateAvailableRequest =
-                String.format("{\"id\": \"myId-1\", \"name\": \"myName-1\", \"endpoint\": \"http://192.168.2.49/ob-myId-1\", \"customerId\": \"%s\", \"status\": \"AVAILABLE\"}",
+                String.format("{\"id\": \"myId-1\", \"name\": \"myName-1\", \"endpoint\": \"http://192.168.2.49/ob-myid-1\", \"customerId\": \"%s\", \"status\": \"AVAILABLE\"}",
```

Comment:
Is this related with the PR?

Reply:
Yes the new `sanitizeName` provided by fabric8 has changed behaviour and makes the string lowercase

Reply:
Just for the record.. 
![Screenshot from 2022-01-17 08-54-45](https://user-images.githubusercontent.com/18282531/149729494-d1fb46e8-7185-4c21-aa0c-75e752d4b4f2.png)
while with the previous version of the library this test fails